### PR TITLE
V6/Fix dapps confirmation layout issues

### PR DESCRIPTION
**Pull request by hsurf22**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
   - Fix dapps confirmation layout issues

### CODE REVIEW

File: src/modules/confirmation/components/TransactionBalance.vue [hunk 1]
```diff
@@ -23,48 +23,77 @@
...
           <span class="searchText--text font-weight-medium">{{
             currency
           }}</span></v-col
         >
         <v-col
           v-if="isEthNetwork"
-          class="py-0 my-0 text-right font-weight-medium"
-          cols="3"
+          class="d-flex align-center pl-6"
+          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
```
Comment (olgakup) [inline src/modules/confirmation/components/TransactionBalance.vue:94]: change this to: class="justify-start justify-sm-end"

File: src/modules/confirmation/components/TransactionBalance.vue [hunk 2]
```diff
@@ -23,48 +23,77 @@
...
+      <v-row dense>
+        <v-col class="d-flex align-center text-uppercase" cols="4">
           <v-icon color="transparent"> mdi-circle-medium </v-icon>
           <h6 class="searchText--text font-weight-medium">Total Pay</h6>
         </v-col>
-        <v-col class="py-0 my-0 text-right font-weight-medium" cols="5">
-          <span>{{ total }} </span>
+        <v-col
+          class="d-flex align-center pl-6"
+          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
```
Comment (olgakup) [inline src/modules/confirmation/components/TransactionBalance.vue:82]: change this to: class="justify-start justify-sm-end"

File: src/modules/confirmation/components/TransactionBalance.vue [hunk 3]
```diff
@@ -23,48 +23,77 @@
...
+          <span class="mr-1">- {{ txFee }}</span>
           <span class="searchText--text font-weight-medium">{{
             currency
           }}</span></v-col
         >
-        <v-col v-if="isEthNetwork" class="py-0 my-0 text-right" cols="3">
+        <v-col
+          v-if="isEthNetwork"
+          class="d-flex align-center pl-6"
+          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
```
Comment (olgakup) [inline src/modules/confirmation/components/TransactionBalance.vue:67]: change this to: class="justify-start justify-sm-end"

File: src/modules/confirmation/components/TransactionBalance.vue [hunk 4]
```diff
@@ -23,48 +23,77 @@
...
+      <v-row dense :class="!$vuetify.breakpoint.smAndUp ? 'mb-5' : ''">
+        <v-col class="d-flex align-center text-uppercase" cols="4">
           <v-icon color="#f5a623"> mdi-circle-medium </v-icon>
           <h6 class="searchText--text font-weight-medium">Transaction Fee</h6>
         </v-col>
-        <v-col class="py-0 my-0 text-right" cols="5">
-          <span> - {{ txFee }} </span>
+        <v-col
+          class="d-flex align-center pl-6"
+          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
```
Comment (olgakup) [inline src/modules/confirmation/components/TransactionBalance.vue:55]: change this to: class="justify-start justify-sm-end"

File: src/modules/confirmation/components/TransactionBalance.vue [hunk 5]
```diff
@@ -23,48 +23,77 @@
...
+          <span class="mr-1">- {{ value }}</span>
           <span class="searchText--text font-weight-medium">{{
             currency
-          }}</span></v-col
+          }}</span>
+        </v-col>
+        <v-col
+          v-if="isEthNetwork"
+          class="d-flex align-center pl-6"
+          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
```
Comment (olgakup) [inline src/modules/confirmation/components/TransactionBalance.vue:40]: change this to: class="justify-start justify-sm-end"

File: src/modules/confirmation/components/TransactionBalance.vue [hunk 6]
```diff
@@ -23,48 +23,77 @@
             height="17"
           />
         </v-col>
-        <v-col class="py-0 my-0 text-right" cols="5">
-          <span> - {{ value }} </span>
+        <v-col
+          class="d-flex align-center pl-6"
+          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
```
Comment (olgakup) [inline src/modules/confirmation/components/TransactionBalance.vue:28]: change this to: class="justify-start justify-sm-end"

File: src/modules/confirmation/components/TransactionBalance.vue [hunk 7]
```diff
@@ -1,20 +1,20 @@
...
     </v-row>
 
-    <mew-progress-bar :balance-obj="progressBar" />
+    <mew-progress-bar class="my-5" :balance-obj="progressBar" />
 
-    <div class="mt-5">
-      <v-row class="py-0">
-        <v-col class="d-flex align-center py-1 my-0 text-uppercase" cols="4">
+    <div>
+      <v-row dense :class="!$vuetify.breakpoint.smAndUp ? 'mb-5' : ''">
```
Comment (olgakup) [inline src/modules/confirmation/components/TransactionBalance.vue:16]: you can state this as: class="mb-sm-5"

**Review by olgakup — CHANGES_REQUESTED** [review]

File: src/modules/confirmation/components/TransactionBalance.vue (hunk 2 above)
Comment (hsurf22) [reply src/modules/confirmation/components/TransactionBalance.vue:82]: fixed

File: src/modules/confirmation/components/TransactionBalance.vue (hunk 4 above)
Comment (hsurf22) [reply src/modules/confirmation/components/TransactionBalance.vue:55]: fixed

File: src/modules/confirmation/components/TransactionBalance.vue (hunk 6 above)
Comment (hsurf22) [reply src/modules/confirmation/components/TransactionBalance.vue:28]: fixed

File: src/modules/confirmation/components/TransactionBalance.vue (hunk 7 above)
Comment (hsurf22) [reply src/modules/confirmation/components/TransactionBalance.vue:16]: fixed

File: src/modules/confirmation/components/TransactionBalance.vue (hunk 5 above)
Comment (hsurf22) [reply src/modules/confirmation/components/TransactionBalance.vue:40]: fixed

File: src/modules/confirmation/components/TransactionBalance.vue (hunk 3 above)
Comment (hsurf22) [reply src/modules/confirmation/components/TransactionBalance.vue:67]: fixed

File: src/modules/confirmation/components/TransactionBalance.vue (hunk 1 above)
Comment (hsurf22) [reply src/modules/confirmation/components/TransactionBalance.vue:94]: fixed

File: src/modules/confirmation/components/ConfirmationTransaction.vue [hunk 8]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-sheet max-width="600px" class="pa-8">
+  <v-sheet max-width="600px" class="pa-4 pa-md-8">
```
[the remark this answers is not in the document]
Comment (hsurf22) [reply src/modules/confirmation/components/ConfirmationTransaction.vue:2]: fixed

File: src/modules/confirmation/components/TransactionBalance.vue [hunk 9]
```diff
@@ -24,8 +24,7 @@
           />
         </v-col>
         <v-col
-          class="d-flex align-center pl-6"
-          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
+          class="d-flex align-center pl-6 justify-start justify-ms-end"
```
Comment (olgakup) [inline src/modules/confirmation/components/TransactionBalance.vue:27]: should it be `justify-sm-end` instead of `justify-ms-end`

**Review by olgakup — CHANGES_REQUESTED** [review]

**Review by olgakup — APPROVED** [review]

**Pull request merged**